kubernetes/multi-tenant/README.md

```diff
@@ -11,17 +11,31 @@ In Kubernetes terminology, a single voice gateway instance equates to a single P
 * Creates a 2 GB persistent volume called recordings to store call recordings
 * Recording is disabled by default. To enable recording set the value of ENABLE_RECORDING variable to true
 
-## To deploy Voice Gateway in multi-tenant mode:
 
-* Configure properties in tenantConfig.json before deployment. For more information - [Configuring multi-tenancy](https://www.ibm.com/support/knowledgecenter/SS4U29/multitenancy.html)
-* Create secret called tenantconfig from the file tenantConfig.json:
 
-```bash
-kubectl create secret generic tenantconfig --from-file=tenantConfig.json
-```
-
-* Deploy the Voice Gateway:
-
-```bash
-kubectl create -f deploy-multitenant.json
-```
\ No newline at end of file
+# Deploying Voice Gateway in multi-tenant mode:
+
+1) Configure properties in tenantConfig.json before deployment. 
+   - For more information - [Configuring multi-tenancy](https://www.ibm.com/support/knowledgecenter/SS4U29/multitenancy.html)
+
+1) Create secret called tenantconfig from the file tenantConfig.json:
+   ```bash
+   kubectl create secret generic tenantconfig --from-file=tenantConfig=tenantConfig.json
+   ```
+
+1) If you want to enable recording (Optional): 
+   - Set ENABLE_RECORDING to true in deploy-multitenant.yaml and create the recording PersistentVolume and PersistentVolumeClaim using the recording-pv.yaml and recording-pvc.yaml files.
```

Comment:
Is there a command people can run to create the volumes? is it `kubectl create -f recording-pv.yaml`?

Reply:
Maybe a link to to the Kubernetes page on PersistentVolumes